[2025-02-13T08:00:16.729+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T08:00:16.748+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Covid19_ETL.transform_data manual__2025-02-13T07:59:06.648644+00:00 [queued]>
[2025-02-13T08:00:16.759+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Covid19_ETL.transform_data manual__2025-02-13T07:59:06.648644+00:00 [queued]>
[2025-02-13T08:00:16.760+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-13T08:00:16.783+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_data> on 2025-02-13 07:59:06.648644+00:00
[2025-02-13T08:00:16.795+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=502) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-13T08:00:16.798+0000] {standard_task_runner.py:72} INFO - Started process 504 to run task
[2025-02-13T08:00:16.798+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Covid19_ETL', 'transform_data', 'manual__2025-02-13T07:59:06.648644+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/covid_19.py', '--cfg-path', '/tmp/tmphft7u748']
[2025-02-13T08:00:16.801+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask transform_data
[2025-02-13T08:00:16.866+0000] {task_command.py:467} INFO - Running <TaskInstance: Covid19_ETL.transform_data manual__2025-02-13T07:59:06.648644+00:00 [running]> on host 3b43358e3e84
[2025-02-13T08:00:17.225+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Covid19_ETL' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T07:59:06.648644+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-13T07:59:06.648644+00:00'
[2025-02-13T08:00:17.228+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-13T08:00:17.256+0000] {warnings.py:112} WARNING - /opt/***/dags/covid_19.py:63: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(raw_data)

[2025-02-13T08:00:17.418+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-13T08:00:17.437+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-13T08:00:17.437+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Covid19_ETL, task_id=transform_data, run_id=manual__2025-02-13T07:59:06.648644+00:00, execution_date=20250213T075906, start_date=20250213T080016, end_date=20250213T080017
[2025-02-13T08:00:17.497+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T08:00:17.519+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-13T08:00:17.526+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
